### PR TITLE
Add abstract

### PR DESCRIPTION
**Pull request by agitter**
I'd like to keep this short and avoid being too redundant with the existing intro.

### CODE REVIEW

File: content/01.abstract.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
 ## Abstract {.page_break_before}
 
-
+Scientific literature reviews are ideal for open, collaborative writing.
+Allowing any interested individual to contribute as an author can strengthen a review, providing broad and fair coverage of the subject matter.
+However, the traditional multi-author writing process breaks down at scale.
+We present techniques for overcoming the challenges of open manuscript writing.
```
Comment (dhimmel) [inline content/01.abstract.md:6]: Very strong first four sentences!

Do we want to mention the deep review for context? Something like "based on our experience with the Deep Review, where XX authors combined expertise to review deep learning in precision medicine.

File: content/01.abstract.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
...
+These include approaches for managing distributed authors and our new software for automating citation and manuscript building.
```
Comment (dhimmel) [inline content/01.abstract.md:7]: I think the abstract needs to mention Manubot by name and provide some more information on it. One point after all is for this article to be the publication citation for the Manubot. I'm happy to propose some text here... probably once we merge this PR (the content that is here is good and mergeable).

**Review by dhimmel — APPROVED** [review]

File: content/01.abstract.md (hunk 1 above)
Comment (agitter) [reply content/01.abstract.md:6]: My inclination was to keep it as short as possible.  I read several feature article abstracts and some were only 3 sentences.

File: content/01.abstract.md (hunk 2 above)
Comment (agitter) [reply content/01.abstract.md:7]: Sounds good.  I'll merge this so you can take a pass over the abstract.

**Pull request merged**